lib/owl2html.js

```diff
@@ -14,7 +14,7 @@ var OwlReader = require('./OwlReader.js');
 function owl2html (triples) {
   var ontology = new OwlReader(triples);
   var jadeTemplate = fs.readFileSync('lib/owl2html.jd', "utf8");
-  var j = jade.compile(jadeTemplate);
+  var j = jade.compile(jadeTemplate, { pretty: true });
```

Comment:
Dammit yes, that was for me for easier debugging :)

Op 30 aug. 2016 10:23 a.m. schreef "Pieter Colpaert" <
notifications@github.com>:

> In lib/owl2html.js
> <https://github.com/mmlab/owlturtle2http/pull/1#discussion_r76750027>:
>
> > @@ -14,7 +14,7 @@ var OwlReader = require('./OwlReader.js');
> >  function owl2html (triples) {
> >    var ontology = new OwlReader(triples);
> >    var jadeTemplate = fs.readFileSync('lib/owl2html.jd', "utf8");
> > -  var j = jade.compile(jadeTemplate);
> > +  var j = jade.compile(jadeTemplate, { pretty: true });
>
> why pretty true? Shouldn't we minify the code for faster load times?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mmlab/owlturtle2http/pull/1/files/6441d146909d0e6ce662636179a6761caca6c4c3#r76750027>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAozpYPlk5oHjd4b1aTzTsmSspZ2TH6Lks5qk-iSgaJpZM4JwO9t>
> .
>